test/sequential/test-fs-readfile.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const common = require('../common');
+
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+// In case openssl is not available
+function slowRandomBytes(len) {
+  const buf = Buffer.alloc(len);
+  for (let i = 0; i < len; i++)
+    buf[i] = Math.random();
```

Comment:
No-op.  `Math.random()` returns a double between 0 and 1 exclusive.  It gets truncated to a byte here, making it equivalent to `buf[i] = 0`.

You probably intended to write `buf[i] = 256 * Math.random()`.  That said...